Doc/glossary.rst

```diff
@@ -1304,6 +1304,11 @@ Glossary
 
       See also :term:`borrowed reference`.
 
+   t-string
+      String literals prefixed with ``'t'`` or ``'T'`` are commonly called
+      "t-strings" which is short for
+      :ref:`template string literals <t-strings>`.  See also :pep:`750`.
```

Comment:
The PEP is the original change proposal, and it's expected to go go out of date. It's good to have a link to it in the docs (e.g. a “see also” in the main docs for the feature), so people can read the motivation and get at the discussion links, but if there's any other info in the PEP that's relevant to readers, it should go in the docs.

```suggestion
      :ref:`template string literals <t-strings>`.
```
(And yes, the current docs aren't too great at this.)